tests/aritmeticops.py.uast

```diff
@@ -8,6 +8,10 @@ Module {
 .  .  Line: 1
 .  .  Col: 1
 .  }
+.  Properties: {
```

Comment:
Actually is in the part of the normalization inside the driver (`parser.go`) so it can be fixed on this same PR. Good catch.